README.md

```diff
@@ -68,8 +68,7 @@ Finding projects depends on how the search API is structured for every hosting
 platform. For example, you can find all `publiccode.yml` on GitHub files by
 searching using the frontend or the API.
 
-* [GitHub Search `filename:publiccode.yml
-  path:/`](https://github.com/search?q=filename%3Apubliccode.yml+path%3A%2F)
+* [GitHub Search `path:/publiccode.yml`](https://github.com/search?q=path%3A%2Fpubliccode.yml&type=code)
```

Comment:
Great catch. This however is not the right result (15 results). For that I think we might need to use a RegEx for searching. What about this: `path:/(^|\/)publiccode\.yml$/`.

https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code

```suggestion
* [GitHub Search `path:/(^|\/)publiccode\.yml$/`](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code)
```